src/milia/api/io.clj

```diff
@@ -39,12 +39,15 @@
 (defn- req+auth
   "Add authorization to options"
   [req]
-  (let [{:keys [temp-token token username password]} *credentials*]
+  (let [{:keys [temp-token token username password]} *credentials*
+        {:keys [auth-token]} req]
     (if (or temp-token token)
       (assoc req
-             :headers {"Authorization" (if temp-token
-                                         (str "TempToken " temp-token)
-                                         (str "Token " token))})
+             :headers {"Authorization" (if auth-token
```

Comment:
the conditional can perhaps be more simply written as:

```clojure
(join " " (if (and temp-token (not auth-token))
            ["TempToken" token]
            ["Token" (or auth-token token)]))
```

given you import `clojure.string :refer [join]`